api/include/opentelemetry/trace/noop.h

```diff
@@ -47,12 +48,14 @@ class NoopSpan final : public Span
 
   bool IsRecording() const noexcept override { return false; }
 
-  Tracer &tracer() const noexcept override { return *tracer_; }
+  SpanContext GetContext() const noexcept override { return span_context_; }
+  //  Tracer &tracer() const noexcept override { return *tracer_; }
```

Comment:
Why is this commented out, instead of deleted?

Reply:
It was originally there, but as I had changed the definiation for span to exclude them, these methods sort of standout. And I don't want to delete them because these are not my codes

Reply:
I don't think it'll be a problem to delete these functions if they're unused. If they need to be added back, someone will add them back. Do you know who initially wrote this code? It should be captured in the file history. You can try contacting them, otherwise it's probably fine to just delete these.